scripts/ci/requirements.txt

```diff
@@ -20,7 +20,7 @@ boto3
 celery
 cgroupspy
 chartkick
```

Comment:
Thats not 2.7?

Reply:
In the setup.py it is `<2.0`:
https://github.com/apache/incubator-airflow/blob/master/setup.py#L163

In the requirements.txt it is latest:
https://github.com/apache/incubator-airflow/blob/master/scripts/ci/requirements.txt#L23

This is a hotfix for now, later we should let the CI install the dependencies using `setup.py`.